spmd/tensor/ops/matrix_ops.py

```diff
@@ -37,19 +38,20 @@ def mm_prop(
     ):
         return mat1_spec
     else:
+        # TODO(anj): Add logging (or something else) to indicate no output sharding
+        # spec specified. If we don't have autoboxing rules specified in the op below,
+        # we will throw an error without a populated `failed_reason`.
         # not local compute, need to rely on auto redistribute, return None
         return None
 
-
 @register_prop_rule("aten.mm.default")
 def mm_rules(op_schema: OpSchema) -> OutputSharding:
-    mat1_spec, mat2_spec = op_schema.args_spec
+    mat1_spec, mat2_spec = op_schema.args_schema
     return OutputSharding(mm_prop(mat1_spec, mat2_spec))
 
-
 @register_prop_rule("aten.addmm.default")
 def addmm_rules(op_schema: OpSchema) -> OutputSharding:
-    input_spec, mat1_spec, mat2_spec = op_schema.args_spec
+    input_spec, mat1_spec, mat2_spec = op_schema.args_schema
```

Comment:
not sure if changing it to `args_schema` triggered the CI failures? i remembered addmm have some scalar inputs (alpha, beta), probably that's in the kwargs instead?